views/student-orgs/list.js

```diff
@@ -91,12 +51,12 @@ export class StudentOrgsView extends React.Component {
   }
 
   state: {
-    orgs: StudentOrgAbridgedType[],
+    orgs: {[key: string]: StudentOrgAbridgedType[]},
```

Comment:
The old type was inaccurate. I didn't change how they're stored, I just fixed the type. I'm not sure why Flow didn't complain before.